roxie/ccd/ccdfile.cpp

```diff
@@ -1782,6 +1805,7 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
                 offset_t endOffset = (endPage+1) << CacheInfoEntry::pageBits;
                 if (inNodeCache && !keyFailed && localFile && !keyIndex)
                 {
+                    //Pass false for isTLK - it will be initialised from the index header
                     keyIndex.setown(createKeyIndex(fileName, localFile->getCrc(), *localFile.get(), fileIdx, false));  // MORE - we don't know if it's a TLK, but hopefully it doesn't matter
```

Comment:
May want to update the comment on this line given the comment on the line above

Reply:
removed